### ISSUE TITLE
Exponential link decay, user feedback

### ISSUE THREAD

**Issue by JediMaster25** [wibyweb/wiby#3]
I've asked the engine to surprise me twice. The first time I got a website with expired certificates and the second a website with a message saying it was no longer maintained. This doesn't give a good first  impression of the engine.

I think the way to avoid this would be that the users are the ones who need to approve which links belong in the server by voting on submissions and there is some decay over time so that if a website doesn't receive enough votes in a given time it loses votes rapidly and gets discarded.

This may require some sort of trust levels for the users, so that they could moderate themselves, by banning trolls and illegal content.

I don't think is a viable option to expect the admin to check all links every so often.

I've just learned about this project and I don't know much about it so I may be way off in my assumptions.

**Comment by wibyweb** [comment]
Thank you for reporting. Long ago, I turned off SSL verification to be nice to people who maybe forget to renew their certs.  Per your issue, I flipped it back on and will see how that goes. 

As for the website with a message that it is no longer being maintained, that is hard to detect unless the webmaster changed the title of the page or the page responded with something other than code 200.  It can take up to two weeks for a dead page to get flagged and automatically removed. SSL failures will now be included again in that. 

If the page still has its original content on it, then the message of "no longer maintained" isn't reason enough to de-index it unless the webmaster removed everything and only left that message, then one could report the URL through the feedback form located in the about section.

A voting system for pages to be indexed isn't something I would want implemented. That opens up a whole new set of issues that would need dealing with.  Private accounts can be created to assign trusted people access to the back-end administrative pages that handle URL submissions.
